src/App.tsx

```diff
@@ -1,8 +1,98 @@
 import React from 'react';
-import './App.css';
+import { observer } from 'mobx-react-lite';
+import { Heading, Text, Box } from 'rebass';
+import partition from 'lodash/partition';
+import { store } from './store';
+import { PendingBets } from './PendingBets';
+import { CompletedBets } from './CompletedBets';
+import { PrimeForm } from './PrimeForm';
+import styled, { createGlobalStyle } from 'styled-components';
 
-const App: React.FC = () => {
-  return null;
-};
+export const Global = createGlobalStyle`
+  * {
+    box-sizing: border-box;
+  }
+`;
 
-export default App;
+const Container = styled(Box)`
+  padding: 20px;
+  max-width: 800px;
+  margin: 0 auto;
+`;
+
+const Header = styled(Box)`
+  width: 70%;
+  margin-bottom: 20px;
+`;
+
+const IntroText = styled(Text)`
+  & + & {
```

Comment:
Reasons:
- to follow the same convention
- to prevent margin collapsing
- to reduce technical debt
```
margin-bottom: 10px;
&:last-of-type{
    margin-bottom: 0;
}
```